client/src/components/Login.js

```diff
@@ -1,14 +1,50 @@
-import React from "react";
+import React, {useState} from "react";
+import api from '../utils/api';
 
-const Login = () => {
+function Login(props) {
   // make a post request to retrieve a token from the api
   // when you have handled the token, navigate to the BubblePage route
+
+  const [err, setErr] = useState('')
```

Comment:
Nice job with the error handling, something you could add if you wanted to is inside the handleChange, you can set the error to blank when they type in a new login/password